test/common/tests.go

```diff
@@ -48,6 +48,16 @@ var (
 				{"A15 - Verify Stateful Set resources have the expected replicas", TestStatefulSetsExpectedReplicas},
 				{"A26 - Verify Sendgrid Credentials Are Configured Properly", TestSendgridCredentialsAreValid},
 				{"C01 - Verify Alerts are not pending or firing apart from DeadMansSwitch", TestIntegreatlyAlertsPendingOrFiring},
+<<<<<<< HEAD
+=======
```

Comment:
these weird characters were added here to help you resolve the conflicts. As part of resolving the conflict you need to remove them.